src/platform/packages/shared/kbn-coloring/src/shared_components/color_mapping/color/color_handling.test.ts

```diff
@@ -168,17 +172,17 @@ describe('Color mapping - color generation', () => {
         assignments: [
           {
             color: { type: 'colorCode', colorCode: 'red' },
-            rule: { type: 'auto' },
+            rules: [], // auto
             touched: false,
           },
           {
             color: { type: 'colorCode', colorCode: 'blue' },
-            rule: { type: 'matchExactly', values: ['blueCat'] },
+            rules: [{ type: 'raw', value: ['blueCat'] }],
```

Comment:
also here, we should probably change that to `rules: [{ type: 'raw', value: 'blueCat' }]` to match correctly what we are looking for

Reply:
Fixed in 85c970f3bdbcf2820a7e7ec23e01f773a3d814e6